packages/odyssey-react/src/components/Table/TableDataCell.tsx

```diff
@@ -30,11 +32,14 @@ const TableDataCell = React.forwardRef<Ref, Props>((props, ref) => {
   const {
     children,
     format,
+    empty,
     ...rest
   } = props;
 
   const componentClass = useCx(
-    format && `is-${tableClass}-${format}`,
+    styles.cell,
+    format && styles[format],
+    empty && styles.empty,
```

Comment:
Not the way they're written. The empty style is very much about conveying that the state table is empty / has no content

![image](https://user-images.githubusercontent.com/87656589/129098990-819a4fc1-e9c7-45db-8743-743374b974a2.png)

Reply:
Thanks - I'm just trying to reason about how the API is setup - If I provide children to a `TableDataCell` **and** and `empty` prop what is the expected behavior and visual output?

Reply:
essentially the screenshot posted above, a background color and padding are added to the cell and the text color is changed. 

This may be another instance where we want to move to composition. On Forge we had an EmptyState component on our backlog that would contain an image and a message to handle this use case.